src/pane-element.coffee

```diff
@@ -156,5 +156,7 @@ atom.commands.add 'atom-pane',
   'pane:split-down': -> @getModel().splitDown(copyActiveItem: true)
   'pane:close': -> @getModel().close()
   'pane:close-other-items': -> @getModel().destroyInactiveItems()
+  'pane:enlarge': -> @getModel().setFlexScale(@getModel().getFlexScale() * 1.1)
+  'pane:shrink': -> @getModel().setFlexScale(@getModel().getFlexScale() / 1.1)
```

Comment:
Also, would you mind moving these methods into the model themselves so that the command handlers just call things like `pane.increaseSize()` and `pane.decreaseSize()`